geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategy.scala

```diff
@@ -35,6 +35,7 @@ import org.locationtech.geomesa.accumulo.index.QueryPlanners.JoinFunction
 import org.locationtech.geomesa.accumulo.index.Strategy._
 import org.locationtech.geomesa.accumulo.iterators._
 import org.locationtech.geomesa.features.SerializationType.SerializationType
+import org.locationtech.geomesa.filter.PropertyLiteral
```

Comment:
Yes, it was covered by import org.locationtech.geomesa.accumulo.filter._ until I moved PropertyLiteral.